sonar-db/src/main/gen-java/org/sonar/db/protobuf/DbCommons.java

```diff
@@ -0,0 +1,818 @@
+          return this;
+        }
+      }
+
+      public Builder mergeFrom(org.sonar.db.protobuf.DbCommons.TextRange other) {
+        if (other == org.sonar.db.protobuf.DbCommons.TextRange.getDefaultInstance()) return this;
+        if (other.hasStartLine()) {
+          setStartLine(other.getStartLine());
+        }
+        if (other.hasEndLine()) {
+          setEndLine(other.getEndLine());
+        }
+        if (other.hasStartOffset()) {
+          setStartOffset(other.getStartOffset());
+        }
+        if (other.hasEndOffset()) {
+          setEndOffset(other.getEndOffset());
+        }
+        this.mergeUnknownFields(other.getUnknownFields());
+        return this;
+      }
+
+      public final boolean isInitialized() {
+        return true;
+      }
+
+      public Builder mergeFrom(
+          com.google.protobuf.CodedInputStream input,
+          com.google.protobuf.ExtensionRegistryLite extensionRegistry)
+          throws java.io.IOException {
+        org.sonar.db.protobuf.DbCommons.TextRange parsedMessage = null;
+        try {
+          parsedMessage = PARSER.parsePartialFrom(input, extensionRegistry);
+        } catch (com.google.protobuf.InvalidProtocolBufferException e) {
+          parsedMessage = (org.sonar.db.protobuf.DbCommons.TextRange) e.getUnfinishedMessage();
+          throw e;
+        } finally {
+          if (parsedMessage != null) {
+            mergeFrom(parsedMessage);
+          }
+        }
+        return this;
+      }
+      private int bitField0_;
+
+      private int startLine_ ;
+      /**
+       * <code>optional int32 start_line = 1;</code>
+       *
+       * <pre>
+       * Start line. Should never be absent
+       * </pre>
+       */
+      public boolean hasStartLine() {
+        return ((bitField0_ & 0x00000001) == 0x00000001);
+      }
+      /**
+       * <code>optional int32 start_line = 1;</code>
+       *
+       * <pre>
+       * Start line. Should never be absent
+       * </pre>
+       */
+      public int getStartLine() {
+        return startLine_;
+      }
+      /**
+       * <code>optional int32 start_line = 1;</code>
+       *
+       * <pre>
+       * Start line. Should never be absent
+       * </pre>
+       */
+      public Builder setStartLine(int value) {
+        bitField0_ |= 0x00000001;
+        startLine_ = value;
+        onChanged();
+        return this;
+      }
+      /**
+       * <code>optional int32 start_line = 1;</code>
+       *
+       * <pre>
+       * Start line. Should never be absent
+       * </pre>
+       */
+      public Builder clearStartLine() {
+        bitField0_ = (bitField0_ & ~0x00000001);
+        startLine_ = 0;
+        onChanged();
+        return this;
+      }
+
+      private int endLine_ ;
+      /**
+       * <code>optional int32 end_line = 2;</code>
+       *
+       * <pre>
+       * End line (inclusive). Absent means it is same as start line
+       * </pre>
+       */
+      public boolean hasEndLine() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1161)